frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1843,9 +1843,16 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                                       linkText: 'Active hours',
                                       canOpenModal: true,
                                       query: {
-                                          kind: NodeKind.WebActiveHoursHeatMapQuery,
+                                          kind: NodeKind.HeatMapQuery,
+                                          source: {
+                                              kind: NodeKind.EventsNode,
+                                              event: '$pageview',
+                                              name: '$pageview',
+                                              math: BaseMathType.UniqueUsers,
+                                          },
                                           properties: webAnalyticsFilters,
                                           dateRange,
+                                          conversionGoal,
```

Comment:
Is this changing anything for your query or was it just for the type checking?

The `conversionGoal` is used when selecting a event as the conversion goal but it must be handled by the query. The `source` is probably going to be ignored as we're using a custom query but I may be wronged there.

<img width="886" alt="image" src="https://github.com/user-attachments/assets/9730bd71-6234-4bc8-8f02-87a98cb0e94c" />